common/ruff.toml

```
@@ -0,0 +1,21 @@
+# Support Python 3.10+
+target-version = "py310"
+
+# Set the maximum line length to 79.
+line-length = 79
+
+# In addition to the default formatters/linters, add these as well.
+extend-select = [
+  # Add the `line-too-long` rule to the enforced rule set. 
+  "E501", 
+  # Add rule that all functions have docstrings.
+  "D",
+  # Add isort to list.
+  "I
```

```suggestion
  "I"
```
Missing a quote here?